### PR TITLE
fix: Missing more translations and add test that checks strings in json

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/__tests__/config/translations.test.ts [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+    const data = fs.readFileSync(file, { encoding: 'utf8', flag: 'r' })
+    let match
+    // eslint-disable-next-line no-cond-assign
+    while ((match = regex.exec(data)) !== null) {
+      if (match[1].trim()) {
+        const includes = translationKeys.includes(match[1])
+        try {
+          expect(includes).toBe(true)
+        } catch (e) {
+          throw new Error(`Something went wrong with in ${file} value ${match[1]}`)
```
Comment (Chef-Cheems) [inline src/__tests__/config/translations.test.ts:40]: Maybe this could be more descriptive message like `Found unknown key ${match[1]} in ${file}`

Comment (memoyil) [reply src/__tests__/config/translations.test.ts]: Done

**Review by Chef-Cheems — APPROVED** [review]
Looks good. I've talked with Bugs about different capitalization in some strings that we have, will have to go through everything to make stuff consistent. But I see that you just fixed the caps to match json file, so that's great 👍

File: src/__tests__/config/translations.test.ts [hunk 2]
```diff
@@ -1,7 +1,45 @@
...
+  const translationKeys = Object.keys(translations)
+
+  function throughDirectory(directory) {
+    fs.readdirSync(directory).forEach((file) => {
+      const absolute = Path.join(directory, file)
+      if (fs.statSync(absolute).isDirectory()) return throughDirectory(absolute)
+      if (absolute.includes('.tsx')) {
+        return files.push(absolute)
+      }
+      return files.length
```
Comment (hachiojidev) [inline src/__tests__/config/translations.test.ts:22]: Why are we returning `files.length`?

File: src/__tests__/config/translations.test.ts [hunk 3]
```diff
@@ -1,7 +1,45 @@
...
+    })
+  }
+
+  throughDirectory('src/views')
+  throughDirectory('src/components')
```
Comment (hachiojidev) [inline src/__tests__/config/translations.test.ts:27]: Is it faster to do it like this versus the entire `src` directory?

File: src/__tests__/config/translations.test.ts (hunk 2 above)
Comment (memoyil) [reply src/__tests__/config/translations.test.ts:22]: The return above returns new array's length so I keep the consistency although that return is not used at the end

File: src/__tests__/config/translations.test.ts (hunk 3 above)
Comment (memoyil) [reply src/__tests__/config/translations.test.ts]: Those are the 2 directories that we have translations, but I changed to "src" to traverse all directories

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**